RHEL/7/input/xccdf/system/software/integrity.xml

```diff
@@ -227,10 +227,6 @@ The base Red Hat platform already includes a sophisticated auditing system that
 can detect intruder activity, as well as SELinux, which provides host-based
 intrusion prevention capabilities by confining privileged programs and user
 sessions which may become compromised.<br />
-In DoD environments, supplemental intrusion detection tools, such as, the McAfee
-Host-based Security System, are available to integrate with existing infrastructure.
-When these supplemental tools interfere with the proper functioning of SELinux, SELinux
-takes precedence.
```

Comment:
@redhatrises 
I would prefer to keep this section of the guide despite of the fact we will have OVAL check for McAfee HBSS, once this PR is merged. IMHO there still might occur cases when the HBSS might interfere with e.g. SELinux. And I would like to keep the SELinux interfere note there.